llarp/router/router.cpp

```diff
@@ -697,7 +697,10 @@ namespace llarp
       connected += _linkManager.NumberOfPendingConnections();
     }
 
-    _rcLookupHandler.ExploreNetwork();
+    const int interval = isSvcNode ? 30 : 5;
+
+    if((Uptime() / 1000) % interval == 0)
```

Comment:
This has the potential to miss lookups, for example if its called with Uptime() = 29999 on one call and Uptime() = 31000 on the next call then it won't get called on either one.

Reply:
that's fine